### PR TITLE
feat: add a srs key loading bench

### PR DESCRIPTION
**Pull request by cryptonemo**
feat: separate aggregation benches from preprocessing

### CODE REVIEW

File: filecoin-proofs/tests/api.rs [hunk 1]
```diff
@@ -1474,3 +1483,1491 @@ fn create_fake_seal<R: rand::Rng, Tree: 'static + MerkleTreeTrait>(
 
     Ok((sector_id, sealed_sector_file, comm_r, cache_dir))
 }
+
+#[test]
+fn test_aggregate_proof_encode_decode() -> Result<()> {
+    // This byte vector is a natively serialized aggregate proof.
+    let aggregate_proof_bytes = vec![
```
Comment (dignifiedquire) [inline filecoin-proofs/tests/api.rs:1490]: probably better to put into its own file and use include_bytes!

File: filecoin-proofs/tests/api.rs [hunk 2]
```diff
@@ -1474,3 +1483,26 @@ fn create_fake_seal<R: rand::Rng, Tree: 'static + MerkleTreeTrait>(
...
+
+    // Re-serialize the proof to ensure a round-trip match.
+    let mut aggregate_proof_bytes2 = Vec::new();
+    aggregate_proof.write(&mut aggregate_proof_bytes2)?;
+
+    assert_eq!(aggregate_proof_bytes.len(), aggregate_proof_bytes2.len());
+    assert_eq!(aggregate_proof_bytes, aggregate_proof_bytes2.as_slice());
+
+    // Note: the native serialization format is more compact than bincode serialization, so assert that here.
+    let bincode_serialized_proof = serialize(&aggregate_proof)?;
```
Comment (dignifiedquire) [inline filecoin-proofs/tests/api.rs:1504]: lets actually weite the numbers as constants in here as well

File: filecoin-proofs/benches/aggregation.rs [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            *POREP_PARTITIONS
+                .read()
+                .expect("POREP_PARTITIONS poisoned")
+                .get(&SECTOR_SIZE_2_KIB)
+                .expect("unknown sector size"),
+        ),
+        porep_id,
+        api_version: ApiVersion::V1_1_0,
+    };
+    let comm_r: [u8; 32] = [5u8; 32];
```
Comment (dignifiedquire) [inline filecoin-proofs/benches/aggregation.rs:43]: no need to specify the type

**Comment by dignifiedquire** [comment]
what's the state here?

**Comment by cryptonemo** [comment]
> what's the state here?

Needs review/approval

**Review by dignifiedquire — APPROVED** [review]

**Pull request merged**